sdk/core/core/src/az_http_policy.c

```diff
@@ -83,26 +83,26 @@ _az_apply_credential(_az_credential* credential, _az_http_request* ref_request)
 }
 
 AZ_NODISCARD az_result az_http_pipeline_policy_credential(
-    _az_http_policy* policies,
-    void* options,
-    _az_http_request* ref_request,
-    az_http_response* out_response)
```

Comment:
We should be using `out_response` naming.

Reply:
I agree